crates/websocket/src/send.rs

```diff
@@ -258,7 +266,13 @@ pub async fn send_local_notifs(
     }
     // Its a post
     None => {
-      if post.creator_id != person.id {
+      // Don't send a notif to yourself
+      if post.creator_id != person.id
+        // And only add to recipients if that person isn't blocked
+        && check_person_block(person.id, post.creator_id, context.pool())
+          .await
+          .is_ok()
```

Comment:
Could you use a variable for this? Its really confusing to have such a long if statement with a comment in the middle.